### PR TITLE
HTTP Auth example

### PR DESCRIPTION
**Pull request by JEuler**
"Addressing" https://github.com/lejard-h/chopper/issues/122

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -61,3 +61,29 @@ final chopper = ChopperClient(
...
+    if (response?.statusCode == 401) {
+      SharedPrefs.localStorage.remove(tokenHeader);
+      // Navigate to some login page or just request new token
+    }
+    return response;
+  },
+]
+```
+
+Of course, the actual implementation may vary based on how the backend API of your project looks like.
```
Comment (stewemetal) [inline faq.md:89]: Provided that this sentece becomes a part of the documentation, I would rephrase it like this:

The actual implementation of the algorithm above may vary based on how the backend API - more precisely the login and session handling - of your app looks like.

Comment (JEuler) [reply faq.md]: Fixed, thank you!
